services/QuillLMS/app/services/vitally_integration/serialize_vitally_sales_organization.rb

```diff
@@ -76,6 +78,32 @@ def diagnostic_rollups
       }
     end
 
+    private def evidence_rollups
+      school_year_start = School.school_year_start(Time.current)
+      last_school_year_start = school_year_start - 1.year
+      school_year_end = school_year_start + 1.year
+      active_students_this_year = active_students(school_year_start)
+      active_students_last_year = active_students(last_school_year_start, school_year_start)
+
```

Comment:
For consistency, extract out all temp variables:
```
evidence_activities_assigned_all_time = evidence_assigned_count(@district)
evidence_activities_assigned_this_year = evidence_assigned_in_year_count(@district, school_year_start, school_year_end)
evidence_activities_assigned_last_year = evidence_assigned_in_year_count(@district, last_school_year_start, school_year_start)
evidence_activities_completed_all_time = evidence_finished(@district).count
evidence_activities_completed_this_year = evidence_completed_in_year_count(@district, school_year_start, school_year_end)
evidence_activities_completed_last_year = evidence_completed_in_year_count(@district, last_school_year_start, school_year_start)
evidence_activities_completed_per_student_this_year = activities_per_student(active_students_this_year, evidence_activities_completed_this_year)
evidence_activities_completed_per_student_last_year = activities_per_student(active_students_last_year, evidence_activities_completed_last_year)
```

Then you have a simpler resulting hash:
```
{
  evidence_activities_assigned_all_time:,
  evidence_activities_assigned_this_year:,
  evidence_activities_assigned_last_year:,
  evidence_activities_completed_all_time:,
  evidence_activities_completed_this_year:,
  evidence_activities_completed_last_year:,
  evidence_activities_completed_per_student_this_year:,
  evidence_activities_completed_per_student_last_year:
}
```